backend/apps/sandbox/services/pipeline_simulator.py

```
@@ -69,15 +70,21 @@ def _simulate_security_logs(code: str) -> tuple[str, str]:
     if re.search(r"os\.system|subprocess\.call", code):
         issues.append("MEDIUM Shell injection risk – use subprocess.run with list args")
     if re.search(r"pickle\.loads?", code):
-        issues.append("HIGH   Unsafe deserialization via pickle – never deserialize untrusted data")
+        issues.append(
+            "HIGH   Unsafe deserialization via pickle – never deserialize untrusted data"
```

Ruff (0.15.21)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)